lib/player.rb

```diff
@@ -0,0 +1,56 @@
+module Scrabble
+  class Player
+
+    attr_reader :name, :plays, :tiles
+
+    def initialize(name)
+      @name = name
+      @plays = []
+      @tiles = []
+    end
+
+    def play(word)
+      if won?
+        return false
+      else
+        plays << word
+        Scoring.score(word)
+      end
```

Comment:
this method forgets to take out the letters from the player's `@tiles` (not necessary for any requirements)